src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -7,21 +7,69 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Reflection;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.Metadata;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using System.Runtime.CompilerServices;
 
 namespace System.Reflection.Emit.Experimental
 {
+
     public class AssemblyBuilder: System.Reflection.Assembly
     {
-        internal AssemblyBuilder() { }
+        private static readonly Guid _s_guid = new Guid("87D4DBE1-1143-4FAD-AAB3-1001F92068E6");//Some random ID, Need to look into how these should be generated.
+        private static readonly BlobContentId _s_contentId = new BlobContentId(_s_guid, 0x04030201);
+        private BlobBuilder _emptyBlob = new BlobBuilder();
+        public override string? FullName { get; }
+        private static MetadataBuilder _metadata = new MetadataBuilder();
+        private static IDictionary<string,ModuleBuilder> moduleStorage = new Dictionary<string, ModuleBuilder>();
```

Comment:
```suggestion
        private static MetadataBuilder _metadata = new MetadataBuilder();
        private static IDictionary<string,ModuleBuilder> moduleStorage = new Dictionary<string, ModuleBuilder>();
```
These two should not be static. Also, `moduleStorage` should have `_` prefix.